PhysicsTools/PatAlgos/python/tools/trigTools.py

```diff
@@ -152,7 +152,7 @@ def toolCode( self, process ):
         trigEvtProdMod                    = getattr( process, triggerEventProducer )
         trigEvtProdMod.processName        = hltProcess
         trigEvtProdMod.patTriggerProducer = cms.InputTag( triggerProducer )
-        if not path is '':
```

Comment:
This is from #34253: a rebase could get rid of these spurious additions in the diff

Reply:
Thanks for spotting, I made some mistakes when moving around directories.